lib/Net/DRI/Protocol/EPP/Extensions/VeriSign/Platforms/NameStore.pm

```diff
@@ -77,10 +77,11 @@ sub setup
 sub default_extensions
 {
  my ($self,$rp)=@_;
- $self->{brown_fee_version} = $rp->{brown_fee_version} if exists $rp->{brown_fee_version};
- my @c=qw/VeriSign::Sync VeriSign::PollLowBalance VeriSign::PollRGP VeriSign::IDNLanguage VeriSign::WhoWas VeriSign::Suggestion VeriSign::Balance GracePeriod SecDNS ChangePoll LaunchPhase VeriSign::DefReg VeriSign::EmailFwd/;
+ $self->{fee_version} = $rp->{fee_version} if exists $rp->{fee_version};
+ # as per MH discussion lets only enforce usage of standard Fee extension to avoid problems (supported by Verisign for all TLDs)
```

Comment:
Can change the comment to:
As VeriSign has started to use the RFC8748 Fee extension (fee-1.0), we are no longer supporting PremiumDomain and CentralNic::Fee to avoid conflicts. These can still be loaded manually in add_current_profile()